packages/orchestration/src/cosmos-api.ts

```diff
@@ -219,3 +224,15 @@ export type IBCMsgTransferOptions = {
   timeoutTimestamp?: MsgTransfer['timeoutTimestamp'];
   memo?: string;
 };
+
+export type CosmosChainAccountMethods<CCI extends CosmosChainInfo> =
+  (CCI extends {
+    icaEnabled: true;
+  }
+    ? IcaAccount
+    : {}) &
+    CCI extends {
+    stakingTokens: {};
```

Comment:
does it matter that `stakingTokens` is an Array? should this be...

```suggestion
    stakingTokens: [];
```

Reply:
Is an array accurate? I know this is how `chain-registry` has it modeled, but I am skeptical. The `bond_denom` field in `x/staking` params seems to indicate just a single entry: https://buf.build/cosmos/cosmos-sdk/docs/main:cosmos.staking.v1beta1#cosmos.staking.v1beta1.Params

Reply:
Yes, it is curious. https://github.com/search?q=repo%3Acosmology-tech%2Fchain-registry+stakingTokens shows only examples of a single element in the array.

I consider this out of scope that will be resolved in already scheduled issues.